test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -110,6 +111,64 @@ export function setup(opts: minimist.ParsedArgs) {
 			await app.code.dispatchKeybinding('escape');
 		});
 
+		describe('Notebook keyboard navigation', async () => {
+			it('can enter and exit edit mode and navigate using keyboard nav', async function () {
+				const app = this.app as Application;
+				await app.workbench.sqlNotebook.newUntitledNotebook();
+				await app.workbench.sqlNotebook.addCellFromPlaceholder('Code'); // add new code cell
+				await app.workbench.sqlNotebook.waitForPlaceholderGone();
+				const activeCodeCellId = (await app.workbench.sqlNotebook.getActiveCell()).attributes['id'];
+				await app.workbench.sqlNotebook.waitForTypeInEditor('code cell', activeCodeCellId); // the new cell should be in edit mode
+
+				await app.workbench.sqlNotebook.addCell('markdown'); // add markdown cell and wait for it to activate
+				await new Promise(c => setTimeout(c, 2000));
+				const activeTextCellId = (await app.workbench.sqlNotebook.getActiveCell()).attributes['id'];
+				await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Split View');
+				await app.workbench.sqlNotebook.waitForTypeInEditor('text cell', activeTextCellId); // Text cell should be in edit mode
+
+				await app.code.dispatchKeybinding('escape'); // exit edit mode and stay in browse mode
+				await app.code.dispatchKeybinding('up'); // select code cell
+				await app.workbench.sqlNotebook.waitForActiveCell(activeCodeCellId); // check that the code cell is now active
+				await app.code.dispatchKeybinding('enter');
+				await app.workbench.sqlNotebook.waitForTypeInEditor('test', activeCodeCellId); // code cell should be in edit mode after hitting enter
+				await app.code.dispatchKeybinding('escape'); // exit edit mode and stay in browse mode
+				await app.code.dispatchKeybinding('down'); // select text cell
+				await app.code.dispatchKeybinding('enter');
+				await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Split View');
+				await app.workbench.sqlNotebook.waitForTypeInEditor('test', activeTextCellId); // text cell should be in edit mode after hitting enter
+
+				await app.code.dispatchKeybinding('escape');
+				await app.code.dispatchKeybinding('escape'); // hitting escape twice deselects all cells
+				await app.workbench.sqlNotebook.waitForActiveCellGone();
+			});
+
+			it('looping from top to bottom using keyboard', async function () {
+				const app = this.app as Application;
+				await app.workbench.sqlNotebook.openFile('untrusted.ipynb');
```

Comment:
It'd be better not to have this dependent on that notebook - we shouldn't be sharing notebooks that are for a specific purpose like this.

I'd suggest just adding the cells you want manually. There isn't really any reason to have a premade notebook for this I think